static/app/views/settings/account/accountEmails.tsx

```diff
@@ -27,10 +29,6 @@ type State = AsyncView['state'] & {
 };
 
 class AccountEmails extends AsyncView<Props, State> {
```

Comment:
does this still need to be an AsyncView if i've moved the only async part of this view into its own component?

Reply:
which way is preferable? is keeping it the way it is okay?

Reply:
It might be misleading to have an AsyncView that isn't fetching anything. 

You can look at other files with `import SentryDocumentTitle from 'app/components/sentryDocumentTitle'` for reference.

Reply:
actually scratch this -- its still making other API calls besides the things i moved out -- needs to be async (updating after adding an email address)